numba/core/interpreter.py

```diff
@@ -17,6 +17,13 @@
 from numba.core.byteflow import Flow, AdaptDFA, AdaptCFA
 from numba.core.unsafe import eh
 
+class _UNKNOWN_VALUE(object):
+
+    def __init__(self, varname):
+        self.varname = varname
```

Comment:
perhaps make `varname` a `_varname` so nothing is messing with it.

Reply:
Done in 8602e774b